config/services.yaml

```diff
@@ -4,6 +4,8 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
+    'env(HEARING_POST_FORM_LINK_TEXT)': ''
```

Comment:
Is this used?

Reply:
Yes, it's used in https://github.com/itk-dev/naevnssekretariatet/pull/245/files#diff-c47b03734cd2b4337b345eeba955637ba790d51fd98979f6d366d4acbdb104e0R225.
I may actually be using an old/deprecated way of setting a default value for an environment variable (f. https://symfony.com/doc/3.4/configuration/external_parameters.html#environment-variables) and the current way to do it is by setting the default value in `.env` (cf. https://symfony.com/doc/current/configuration.html#env-file-syntax).

I'll clean this up.

Reply:
Fixed in https://github.com/itk-dev/naevnssekretariatet/pull/245/commits/bde36063f88759cfedee249566f7368c6e4a8571